modules/nf-core/metaphlan/metaphlan/main.nf

```diff
@@ -23,12 +23,13 @@ process METAPHLAN_METAPHLAN {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def input_type  = ("$input".endsWith(".fastq.gz") || "$input".endsWith(".fq.gz")) ? "--input_type fastq" :  ("$input".contains(".fasta")) ? "--input_type fasta" : ("$input".endsWith(".bowtie2out.txt")) ? "--input_type bowtie2out" : "--input_type sam"
+    def input_type  = ("$input".endsWith(".fastq.gz") || "$input".endsWith(".fq.gz")) ? "--input_type fastq" :  ("$input".contains(".fasta")|| "$input".endsWith(".fa.gz")) ? "--input_type fasta" : ("$input".endsWith(".bowtie2out.txt")) ? "--input_type bowtie2out" : "--input_type sam"
```

Comment:
Are `fasta` and `.fa.gz` the only file suffixes allowed for MetaPhlAn4 for fasta input??

Reply:
Good point! It also includes `fasta.gz`

Reply:
OK, maybe there could be a way to simplify this... people could also supply `fna` or `fna.gz` as welll...

Reply:
Yes much nicer! I guess it just missing the `gz` variants (if supported by the tool?)

Reply:
I removed the $ at the end of the searching pattern now, so it should include gz variabts.